homeassistant/components/xiaomi_aqara.py

```diff
@@ -39,6 +39,19 @@ def _validate_conf(config):
             raise vol.Invalid('Invalid key %s.'
                               ' Key must be 16 characters', key)
         res_gw_conf['key'] = key
+
+        host = gw_conf.get('host')
+        if host is not None:
+            res_gw_conf['host'] = host
+            port = gw_conf.get('port')
```

Comment:
You can change this to port = gw_conf.get('port', 9898)

And remove the `if port....`

Reply:
Fixed.